src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -92,23 +101,27 @@ static void LaunchAndWait(Permutation permutation, TestDescriptor testDescriptor
             var permutationArgs = PermutationParser.ToArgs(permutation);
             var sessionIdArgument = $" --sessionId={SessionId}";
 
+            //var useShellExecute = Environment.OSVersion.Platform != PlatformID.Win32NT;
             ProcessStartInfo pi;
             if (permutation.Platform == Platform.NetFramework)
             {
-                var processId = DebugAttacher.GetCurrentVisualStudioProcessId();
-                var processIdArgument = processId >= 0 ? $" --processId={processId}" : string.Empty;
-
+                // ReSharper disable once RedundantAssignment
+                var processIdArgument = string.Empty;
+#if NET452
+                var processId = VisualStudioDebugHelper.DebugAttacher.GetCurrentVisualStudioProcessId();
+                processIdArgument = processId >= 0 ? $" --processId={processId}" : string.Empty;
+#endif
                 pi = new ProcessStartInfo(testDescriptor.ProjectAssemblyPath, permutationArgs + sessionIdArgument + processIdArgument)
                 {
-                    UseShellExecute = false,
+                    UseShellExecute = true,
```

Comment:
I played around with those values again, I noticed the following:
* it doesn't seem to make a difference when running on the full framework.
* it doesn't seem to make a difference when running on .net core on a linux box
* when running the .net core tests via R#, it doesn't show the endpoint console when this is set to false.

based on that, I'd propose to stick with `true` for now unless there is some other problem caused by this?